src/swap/v2/constants.ts

```diff
@@ -25,3 +25,7 @@ export const V2_SWAP_APP_CALL_SWAP_TYPE_ARGS_ENCODED = {
   [SwapType.FixedInput]: encodeString("fixed-input"),
   [SwapType.FixedOutput]: encodeString("fixed-output")
 } as const;
+
+export const V2_SWAP_ROUTER_APP_ARGS_ENCODED = {
+  ASSET_OPT_IN: encodeString("asset_opt_in")
+};
```

Comment:
```suggestion
} as const;

export const V2_SWAP_ROUTER_APP_ARGS_ENCODED = {
  ASSET_OPT_IN: encodeString("asset_opt_in")
} as const;
```
I think `as const` makes it type-aware, otherwise it will be `Record<string, any>`